pkg/apis/operator/v1alpha1/tektonpipeline_lifecycle.go

```diff
@@ -21,13 +21,25 @@ import (
 	"knative.dev/pkg/apis"
 )
 
+const (
+	CrdInstalled         apis.ConditionType = "CrdInstalled"
+	ClustersScoped       apis.ConditionType = "ClusterScopedInstalled"
+	NamespaceScoped      apis.ConditionType = "NamespaceScopedInstalled"
+	DeploymentsInstalled apis.ConditionType = "DeploymentInstalled"
+	WebhookReady         apis.ConditionType = "WebhookReady"
+	ControllerReady      apis.ConditionType = "ControllerReady"
+)
```

Comment:
```suggestion
	CrdInstalled         apis.ConditionType = "CRDsInstalled"
	ClustersScoped       apis.ConditionType = "ClusterScopedResourcesInstalled"
	NamespaceScoped      apis.ConditionType = "NamespaceScopedResourcesInstalled"
	DeploymentsInstalled apis.ConditionType = "DeploymentsAvailable"
	WebhookReady         apis.ConditionType = "WebhooksReady"
	ControllerReady      apis.ConditionType = "ControllersReady"
)
```

So that we can make our Status more `read optimized and meaningful` for our users